uint32 id
time stamp
uint16 Module_Voltage                                  
uint16 Module_Capacitance_Temperature                        
uint16 Module_Board_Temperature
uint16 Module_Voltage_Overvoltage_Abnormal
uint16 Module_Capacity_Abnormal
uint16 other_data
uint16 Module_Voltage_Overvoltage_Warming                          

